src/colors.css

```diff
@@ -58,4 +58,69 @@ html[data-color-scheme="dark"] {
   color-scheme: dark;
 }
 
-/** TODO: Add colors for high contrast modes */
+html[data-color-scheme="light-high-contrast"], html:not([data-color-scheme]) {
+  --color-neutral05: #fff;
+  --color-neutral10: #fff;
+  --color-neutral15: #fff;
+  --color-neutral20: #000;
+  --color-neutral25: #000;
+  --color-neutral30: #000;
+  --color-neutral40: #000;
+  --color-neutral50: #000;
+  --color-neutral60: #000;
+  --color-neutral70: #000;
+  --color-neutral80: #000;
+  --color-neutral90: #000;
+
+  --color-danger0: #a5102d;
+  --color-danger1: #a5102d;
+  --color-danger2: #a5102d;
+  --color-danger4: #a5102d;
+  --color-danger5: #a5102d;
```

Comment:
This color is from the editor too, I assume? I would suggest instead using the same red tone as the light mode above. With the same lightness as this color (35), that would be: `#a50613`.

Though I'm also wondering: danger0, 1 and 2 are very bright and "rounding" them would probably result in white rather than red? They are probably only used as background anyway with dark text on top of them. So yeah, is white better here?

Reply:
Yup, copied this color straight from the editor. The color you suggested should be good too, as long as it keeps a high contrast (didn't check).

Whether the lighter colors should be white instead of red: Hard for me to say, probably depends on how they are intended to be used?  I assumed that all colors named "danger" should signal danger, so I made them all red. If you think they're better of white we can go with that too.

Reply:
Ah, I can totally see that line of thought. 0 and 1 are certainly just used as background that's slightly colored. The slight coloring won't be seen by people who rely on high contrast themes, yes. So if we make it white and no other thing of the element signals danger, then that semantic is completely lost, which is bad. But if we make it the same strong red, then the foreground (text) needs to change color to white instead. So in either case, we have to do additional work to make that element work out. And I think I would prefer having a white background with some other red parts, instead of having a dark red background and white text. The former fits better into the rest of the scheme and closer to the non-high-contrast design. (Everything was formulated for light mode; for dark mode, the same logic applies with some words inverted)

Reply:
If that makes the most sense for lighter danger colors then sure, let's go with that. So which ones should go white? 0,1 or 0,1,2?

Reply:
Stop asking so many difficult questions :grimacing: Ehm... maybe just 0 and 1 for now. We will have to revisit this in the future anyway.